### ISSUE TITLE
Own local MQTT broker

### ISSUE THREAD

**Issue by CandymanCZ** [fischertechnik/txt_training_factory#13]
Hello,
I have the Factory Simulation 9V and wanted to connect it to the cloud. As I read into the matter a little bit, I discovered that I need the Factory Industry 4.0 to have the cloud functionality. As I analyzed all the programs and how interconnected they are, I discovered that the only thing keeping me from connecting the model to the cloud is the missing camera module, which all the sensor data is sent to through a local MQTT connection, which is then sent to the cloud. My question is: Is there a way I could substitute the local MQTT server with my own? Maybe to trick all the TXT controllers into sending all their data to a different broker adress?

**Comment by CvanLeeuwenFt** [comment]
Yes it possible to run this factory in a local network with the NodeRed dasboard also running on a laptop in this local network.
But you need to make some small changes in the settings.
This is out of the scope of this github entry.
See also the ft-forum and/or contact me by PM.

**Comment by CandymanCZ** [comment]
> Yes it possible to run this factory in a local network with the NodeRed dasboard also running on a laptop in this local network. But you need to make some small changes in the settings. This is out of the scope of this github entry. See also the ft-forum and/or contact me by PM.

Thank you, now there is a glimmer of home for me. I have looked through the NodeRed documentation for this before but I still was not able to determine if the TXT with the camera (which is normally the centerpoint of MQTT connection for the whole factory) could be completely omitted with NodeRed.
